xray/utils/resultstable.go

```diff
@@ -419,6 +420,9 @@ func (s *severity) printableTitle(isTable bool) string {
 	if isTable && log.IsStdOutTerminal() && log.IsColorsSupported() {
 		return s.style.Render(s.emoji + s.title)
 	}
+	if os.Getenv("GITLAB_CI") != "" {
+		return s.style.Render(s.emoji + s.title)
+	}
```

Comment:
Looks like we return the same values here and above. Let's unite the 2 ifs.

Reply:
Done!